### PR TITLE
Twin sample: Synchronous and no user prompt

### PR DESCRIPTION
**Pull request by momuno**

### CODE REVIEW

File: sdk/samples/iot/hub/src/paho_iot_hub_twin_example.c [hunk 1]
```diff
@@ -141,37 +171,9 @@ int main()
...
-      }
-      if (input == 'q')
-      {
-        break;
-      }
-    }
-  }
+  get_twin_document();
+  send_reported_property();
+  get_desired_property();
```
Comment (danewalton) [inline sdk/samples/iot/hub/src/paho_iot_hub_twin_example.c:176]:
```suggestion
  receive_desired_property();
```
"get" sounds as though it is actively asking for it and expecting something in return. "receive" makes it more explicit it is waiting for something.

Comment (momuno) [reply sdk/samples/iot/hub/src/paho_iot_hub_twin_example.c:176]: totally.

**Comment by danewalton** [comment]
that took a bit of effort man haha

**Pull request merged**